framework/src/modules/chain/init_steps/init_logic_structs.js

```diff
@@ -16,13 +16,9 @@
 
 module.exports = async ({ ed, schema, components: { storage, logger } }) => {
 	const Block = require('../logic/block.js');
-	const Account = require('../logic/account.js');
+	const Account = require('../rounds/account.js');
 
-	const accountLogic = await new Promise((resolve, reject) => {
-		new Account(storage, schema, logger, (err, object) => {
-			err ? reject(err) : resolve(object);
-		});
-	});
+	const accountLogic = new Account(storage, schema, logger);
```

Comment:
would be nice if we can remove from here, and instantiate within the rounds folder